OpenRA.Mods.Common/Widgets/ViewportControllerWidget.cs

```diff
@@ -168,12 +168,27 @@ public override bool YieldKeyboardFocus()
 
 		public override bool HandleKeyPress(KeyInput e)
 		{
-			switch (e.Key)
+			var key = Hotkey.FromKeyInput(e);
+			var ks = Game.Settings.Keys;
+
+			if (key == ks.MapScrollUp)
+			{
+				keyboardDirections = keyboardDirections.Set(ScrollDirection.Up, e.Event == KeyInputEvent.Down); return true;
+			}
```

Comment:
Its not technically a single line statement because of "return;". Should I make it so return is on its own line?